overrides/config/betterquesting/DefaultQuests.json

```diff
@@ -123224,7 +123264,7 @@
           "repeat_relative:1": 1,
           "ismain:1": 0,
           "name:8": "Better than Dirt",
-          "desc:8": "Much better than Dirt in fact...! Craft some additional Elemental Soil and upgrade some batches of them to all 4 different types of Elemental Soil:\n\n- Drop it in Water to upgrade it to Aqueous Soil,\n- Drop it in Lava to upgrade it to Magmatic Soil,\n- Drop it above Y\u003d90 to upgrade it to Caelic Soil,\n- Drop it below Y\u003d30 to upgrade it to Terran Soil.\n\nEach type of Soil has its benefits (read their tooltip).\n\nNote: The effect of the Aqueous Soil can be disabled by placing a piece of Gravel underneath the Soil.\nNote: Aqueous Soil is particularly good for early crop farms, before you have unlocked farmers. Use a Hopper Minecart or the Roots Ritual to collect the drops."
+          "desc:8": "Much better than Dirt in fact...! Craft some additional Elemental Soil and upgrade some batches of them to all 4 different types of Elemental Soil:\n\n- Drop it in Water to upgrade it to Aqueous Soil,\n- Drop it in Lava to upgrade it to Magmatic Soil,\n- Drop it above Y\u003d90 to upgrade it to Caelic Soil,\n- Drop it below Y\u003d30 to upgrade it to Terran Soil.\n\nEach type of Soil has its benefits (read their tooltip).\n\nNote: The effect of the Aqueous Soil can be disabled by placing a piece of Gravel underneath the Soil.\nNote: Aqueous Soil is particularly good for early crop farms, before you have unlocked farmers. The Aqueous Soil drops outputs into inventories below it which can make collecting drops easier and redstone pulses applied to the soil speeds up the crop growth dramatically. Some good options to make good use of the Aqueous Soil include using 2x1 Drawers connected to a Drawer Controller to extract the drops, Hopper Minecarts, and the Roots Ritual. Any sort of redstone timer or clock are a good way to give redstone pulses to the soil."
```

Comment:
the notes would flow better if merged - current it reads "`gravel stops feature. introducing the feature`"
run on sentence in "`The Aqueous Soil drops outputs into inventories below it which can make collecting drops easier and redstone pulses applied to the soil speeds up the crop growth dramatically.`"
also, personal testing doesnt indicate that this happens - redstone burnout clock doesnt seem to grow statistically faster.

Reply:
I went and tested with an rftools timer and could see a significant benefit. Others have also seen benefits when redstone ticking it.

Reply:
the information paragraph still needs more work